graylog2-web-interface/src/views/components/widgets/reexecuteSearchTypes.ts

```diff
@@ -51,8 +52,9 @@ const reexecuteSearchTypes =
       const { result: searchResult } = searchExecutionResult;
       const updatedSearchTypes = searchResult.getSearchTypesFromResponse(searchTypeIds);
       const { result } = selectSearchExecutionResult(getState());
+      const newSearchResult = new SearchResult({ ...result.result, errors: searchResult.result.errors });
 
-      return { result: result.updateSearchTypes(updatedSearchTypes), widgetMapping: view.widgetMapping };
+      return { result: newSearchResult.updateSearchTypes(updatedSearchTypes), widgetMapping: view.widgetMapping };
```

Comment:
The issue was that we just extended the existing `SearchResult`(from `selectSearchExecutionResult(getState())`) with the new search type data, but the error is no longer part of the search type data, it is part of the `errors` array in the new `SearchResult`  (coming from `searchExecutionResult`).

I had to decide if it is a good idea to merge the previous and new search result errors and favoured always using the latest `errors`.